README.md

```diff
@@ -11,23 +11,42 @@ that makes the code less readable.
 
 Add a `#[momo]`  annotation from this crate to split your function
 into an outer conversion and a private inner function.  In return,
-you get some compile time for a tiny bit of runtime  (if at all) –
+you get some compile time for a tiny bit of runtime (if at all) –
 without impairing readability.
 
-For now, the only place where we can put the `#[momo]` annotations
-is on plain functions.
+Conversions currently supported are `Into` (`.into()`), `AsRef`
+(`.as_ref()`), and `AsMut` (`.as_mut()`).  See `enum Conversions`
+in code.
+
+
+## Notes on watt
 
 This new updated version usesi D. Tolnay's [watt] runtime to speed
```

Comment:
I think I made a typo at "uses". Not related to this PR, but appreciated if you're up for a drive-by fix.